lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua

```diff
@@ -4211,6 +4211,7 @@ end
 
 local debugColor = Color (192, 192, 192)
 function Carrier.Debug (message)
+	if not Carrier.IsLocalDeveloperEnabled () then return end
```

Comment:
This is computer generated code. You have to rerun Carrier.Bootstrap.Generator.App

Reply:
I could not figure out how to do this. Please elaborate.

Reply:
`luajit path/to/lua/carrier/bootstrap.luajit.lua Carrier.Bootstrap.Generator.App`